### ISSUE TITLE
Initial thoughts?

### ISSUE THREAD

**Issue by RichardLitt** [RichardLitt/translate-readmes#1]
So, right now, most README.md files are in English. 

What if they weren't? 
### How to fix this
- Translate them all by hand. Long, arduous, morale-sapping.
- Translate them using Machine translation
- Use a hybrid MT/crowd model.
- Don't bother.
### Issues
- Is translation actually a priority?
- Does translated documentation lead to more usage?
- Would coding communities in other languages benefit from translations?

**Comment by richardj** [comment]
A few side notes to consider:
- Do we as technical users of git / github need it, or is it something that applies more to the market we work for than for ourselves?
- Is the idea of English as a main language the correct one with more people speaking Chinese, Hindi.
- What should be the end goal? As most of the technical world (seen from a European perspective in my case) is in English and everything that is not information in my mother tongue or in English is information I just don't bother with.

**Comment by RichardLitt** [comment]
> Do we as technical users of git / github need it, or is it something that applies more to the market we work for than for ourselves?

"We" are not a single group. Developers have different backgrounds - you yourself had to learn English to have this conversation, whereas I was born into it. If language interferes with learning something, and if we can make that easier through translation, then it applies. 

> Is the idea of English as a main language the correct one with more people speaking Chinese, Hindi.

No. It is the _de facto_ web language, though, and I would like to stop sharding of the internet into different language groups if possible. We could also take Chinese repos and translate them. I get a daily email from GitHub about top starred repositories - there's a Chinese one almost every day. 

> What should be the end goal? 

Enabling interaction over code with users of different languages. 

This may not be feasible!

**Comment by richardj** [comment]
> "We" are not a single group. Developers have different backgrounds - you yourself had to learn English to have this conversation, whereas I was born into it. If language interferes with learning something, and if we can make that easier through translation, then it applies.

That is true, but we have a common understanding that most of our documentation that we are used to is in English, so if you want to be efficient on a technical level it is required. With this in mind, would it make any sense to translate everything while every command that is used in a terminal is in English and can't be localized or would make in that case for a lot of ambiguity?

> No. It is the de facto web language, though, and I would like to stop sharding of the internet into different language groups if possible. We could also take Chinese repos and translate them. I get a daily email from GitHub about top starred repositories - there's a Chinese one almost every day.

That is an interesting point, as I don't get such an email thus Chinese repositories for instance are something that I just do not run into. The idea of translating them would make sense, but a danger is the sharding into different language groups. So how would we enable users without a matching language to be able to work together with a readme as a starting point? 

I suppose we could take that as a starting point: What is needed for people without a matching language ability to be able to at least understand, fork and translate existing work.

**Comment by RichardLitt** [comment]
> With this in mind, would it make any sense to translate everything while every command that is used in a terminal is in English and can't be localized or would make in that case for a lot of ambiguity?

Probably not. It might be useful to translate some higher level things, though, like Readme.md's, to help people know if they should put in the time to use the code or not. 

> The idea of translating them would make sense, but a danger is the sharding into different language groups. 

This is already happening; translating would help close this gap, I think? 

> So how would we enable users without a matching language to be able to work together with a readme as a starting point?

The main README.md file could be for the default language for that repo. For other languages, we could have multiple readmes?
